src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -154,29 +164,23 @@ trait CromIamApiService extends Directives with SprayJsonSupport with DefaultJso
 
   def backendRoute: Route = generalGetRoute("backends")
```

Comment:
Is `backends` not intended to be public?

Reply:
It starts with `api` (and I didn't change that)

FWIW the guidance I was given the other day was that it is suggested strongly that the only non-authN'd endpoint for any service is the status endpoint. We don't conform to that suggestion and will need to evaluate, however making the situation worse w/o pondering doesn't seem wise.